MVC/app.js

```diff
@@ -1,14 +1,27 @@
 const express = require('express')
 const app = express()
-
-
-// my different cb functions form controllers
-const {getTopics, beOnApi} = require('./controllers')
+const {getTopics, getApiEndPoints, getArticleById} = require('./controllers')
 
 app.use(express.json())
```

Comment:
Has this line of code been driven by a test? Every code written shoul dbe driven by a test or part of a test but we haven't yet written a test that requires express.json() yet

Reply:
oh that is just something I always put as a default syntax, I just noticed as well none of the other changes I did like erased comments seems to be visible